microsetta_private_api/repo/removal_queue_repo.py

```diff
@@ -14,6 +16,31 @@ def _check_account_is_admin(self, admin_email):
 
             return False if count == 0 else True
 
+    def _row_to_removal(self, r):
+        return RemovalQueueRequest(r['id'], r['account_id'], r['email'],
```

Comment:
In case this is called from a subclass, this is a little more defensive:

```suggestion
        return self.__class__(r['id'], r['account_id'], r['email'],
```